iree/test/e2e/xla_ops/convolution.mlir

```diff
@@ -402,3 +402,27 @@ func @conv2d_1452x2223_dilated_valid() {
        [-1.122044,   -0.41301775, -1.5628793 ]]]]> : tensor<1x2x4x3xf32>) : tensor<1x2x4x3xf32>
   return
 }
+
+func @depthwise_conv() {
```

Comment:
What about making the name more explicit, like adding `_non_1_channel_multiplier` suffix?

Reply:
good idea, thanks!